raiden/tasks.py

```diff
@@ -476,16 +473,84 @@ def _run(self):
                 pex(transfer.target),
             )
 
+        maximum_expiration = (
+            originating_channel.settle_timeout +
```

Comment:
Please rewrite as:

```python
 maximum_expiration = (
            originating_channel.settle_timeout
            + raiden.chain.block_number()
            - 2 # decrement as a safety measure to avoid limit errors
)
```

Much easier to read when skimming through code when the operand is right on the left of the object.